x-pack/plugins/files/common/types.ts

```diff
@@ -132,28 +132,39 @@ export type UpdatableFileAttributes<Meta = unknown> = Pick<FileJSON<Meta>, 'meta
 
 // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
 export type FileShareSavedObjectAttributes = {
-  created_at: string;
+  created: string;
+
+  /**
+   * Secret token used to access the associated file.
+   */
+  token: string;
```

Comment:
I was thinking the token value would be the ID of these saved object. WDYT?

```suggestion
```

Reply:
That was the initial implementation :) 

I added a comment here: https://github.com/elastic/kibana/pull/137172#discussion_r929872540

Let me know what you think!